src/components/patterns/elements/select-list.tsx

```diff
@@ -116,7 +116,7 @@ const SelectList = ({options, label, multiple, ariaLabelledby, ...props}: Props)
     >
       <button
         {...getButtonProps()}
-        className="w-full rounded-3xl border border-black-40 p-5 pl-15 text-left"
+        className={"w-full border border-black-40 px-5 py-9 pl-15 text-left " + (optionChosen ? "rounded-3xl" : "rounded-full")}
```

Comment:
@pookmish Is there a way to differentiate between the select on the homepage and use `rounded-full` there but have this condition on the places to study page?

Reply:
You can add a prop like `rounded` and then pass that prop when you want it to be rounded.
Here's an example of a `borderless` prop for press: https://github.com/SU-SWS/press-nextjs/blob/1.x/src/components/elements/select-list.tsx#L117

Then when you use the select component, you'd just do `<Select .... rounded />`